jerry-main/main-unix.c

```diff
@@ -276,6 +286,14 @@ main (int argc,
     {
       jerry_port_default_set_abort_on_fail (true);
     }
+    else if (!strcmp ("--no-prompt", argv[i]))
+    {
+      no_prompt = true;
+    }
+    else if (!strcmp ("-", argv[i]))
```

Comment:
That is true but we could also simply add a length check to the next `else if` imho. However I'm not really sure which one is better.